src/db.rs

```diff
@@ -1996,9 +1988,27 @@ mod tests {
         assert_iterator(
             &mut iter,
             vec![
-                RowEntry::new_value(&[b'a'; 32], &[b'j'; 32], 0).with_create_ts(0),
-                RowEntry::new_tombstone(&[b'b'; 32], 0).with_create_ts(0),
-                RowEntry::new_value(&[b'c'; 32], &[b'l'; 32], 0).with_create_ts(10),
+                RowEntry {
+                    key: Bytes::copy_from_slice(&[b'a'; 32]),
+                    value: ValueDeletable::Value(Bytes::copy_from_slice(&[b'j'; 32])),
+                    seq: 1,
+                    create_ts: Some(0),
+                    expire_ts: None,
+                },
+                RowEntry {
+                    key: Bytes::copy_from_slice(&[b'b'; 31]),
+                    value: ValueDeletable::Tombstone,
+                    seq: 2,
+                    create_ts: Some(0),
+                    expire_ts: None,
+                },
+                RowEntry {
+                    key: Bytes::copy_from_slice(&[b'c'; 32]),
+                    value: ValueDeletable::Value(Bytes::copy_from_slice(&[b'l'; 32])),
+                    seq: 3,
+                    create_ts: Some(10),
+                    expire_ts: None,
+                },
```

Comment:
Why do we need this change?

Reply:
oh sorry it's a mistake from git conflict resolvement, let me change it back

Reply:
fixed in c4d6860